tasks.rb

```diff
@@ -4484,24 +4484,6 @@ def display_usage(command_list)
   download_epws
 end
 
-if ARGV[0].to_sym == :update_version
```

Comment:
Why remove this?

Reply:
It just seemed useless, there's only a single file to update.